agenta-web/src/components/EvaluationTable/ABTestingEvaluationTable.tsx

```diff
@@ -294,8 +295,10 @@ const ABTestingEvaluationTable: React.FC<EvaluationTableProps> = ({
                             ? testsetRowToChatMessages(evaluation.testset.csvdata[rowIndex], false)
                             : [],
                     )
-                    if (typeof result !== "string") {
+                    if (isFuncResponse(result)) {
                         result = result.message
+                    } else if (isBaseResponse(result)) {
+                        result = result.data.message
```

Comment:
Mmmh, `message` doesn't necessarily exist in `data` and `data` can be anything from a string to an object.